env/ethereum-classic.env

```diff
@@ -3,10 +3,10 @@
 ###############
 REACT_APP_BLOCKCHAIN_URI=https://etc.rivet.link
 REACT_APP_CHAIN_ID=61
-REACT_APP_DJED_ADDRESS=0x847CceD1eF8Dde95FaB8Eb11B281aAA23Dd014D6
+REACT_APP_DJED_ADDRESS=0x621e39a1A4e64f6C3279e55d6EAc97B7Ee8A37b5
 REACT_APP_FEE_UI=0
 REACT_APP_UI=0x0249a1a7778BD203206d1B7b614fB35Ce7f8F285
-REACT_APP_EXPLORER=https://blockscout.com/etc/mainnet/
+REACT_APP_EXPLORER=https://etc.blockscout.com/
```

Comment:
Could you change this line to:

`REACT_APP_EXPLORER=https://etc.blockscout.com`

Reply:
Otherwise the last slash will cause problems in the address.

Reply:
See my other comments.

Whatever you decide to do, ensure that it works with the other env files too (for milkomeda, sepolia, ...).

Reply:
Sure, I have fixed it. Somehow despite the slash, the browser seems to correctly open it, so couldn't detect that.